ddl/ddl.go

```diff
@@ -201,28 +198,29 @@ type DDL interface {
 
 // ddl represents the statements which are used to define the database structure or schema.
 type ddl struct {
-	m sync.RWMutex
+	m          sync.RWMutex
+	infoHandle *infoschema.Handle
+	quitCh     chan struct{}
 
-	infoHandle   *infoschema.Handle
-	hook         Callback
-	hookMu       sync.RWMutex
+	*ddlCtx
+	workers []*worker
+}
+
+type ddlCtx struct {
+	uuid         string
 	store        kv.Storage
 	ownerManager owner.Manager
 	schemaSyncer SchemaSyncer
-	// lease is schema seconds.
-	lease        time.Duration
-	uuid         string
 	ddlJobCh     chan struct{}
 	ddlJobDoneCh chan struct{}
 	ddlEventCh   chan<- *util.Event
-	// reorgCtx is for reorganization.
-	reorgCtx *reorgCtx
+	lease        time.Duration // lease is schema seconds.
 
-	quitCh chan struct{}
-	wait   sync.WaitGroup
+	// hook may be modified.
+	hook   Callback
+	hookMu sync.RWMutex
 
-	workerVars      *variable.SessionVars
-	delRangeManager delRangeManager
```

Comment:
This is useless now?

Reply:
No，we use it in the structure of `worker`.